German completion patch

ArkfrostLumas opened the pull request:
Phew. That was nice, with the new Atom plugin :D 
Everything went smoothly and it didn't take as long as expected. Look at this patch after you've finished my other pull request :) 

- I've changed a few TTS for better pronounciation
- I've added all the missing translations I've found

What's left?:
- o1s-o4s translations
- rabanastre translation

If you find any issues, I'm happy to fix them :3

quisquous commented on ui/raidboss/data/triggers/byakko-ex.js:
```diff
@@ -6,36 +6,55 @@
     {
       id: 'ByaEx Heavenly Strike',
       regex: / 14:27DA:Byakko starts using Heavenly Strike on (\y{Name})/,
+      regexDe: / 14:27DA:Byakko starts using Himmelszorn on (\y{Name})/,
```
Did you mean to include all the byakko stuff in this patch too or is this intended to just be one megapatch and I should close the other one?

quisquous replied:
I'm not sure what you mean.  You said different things in different patches.  It looks like you fixed things here, so maybe we should close the other patch and just have this patch?

ArkfrostLumas replied:
Hmmmm... maybe that might work. I'm not very sure, because:
- with the byakko-translations I did some changes on my local byakko.js and eureka.js only. I uploaded these as byakko-translations branch afterwards
- with the german-completion-patch I did changes afterwards on top of that byakko-translations branch
So if github compares all lines and replaces the difference you can simply close the other pull request and ignore it. If it only changes what was changed within this branch and it's edit you'd have to merge the other one first though.

quisquous commented on ui/raidboss/data/triggers/o7s.js:
```diff
@@ -110,6 +110,7 @@
       id: 'O7S Orb Marker',
       regex: /1B:........:(\y{Name}):....:....:0017:0000:0000:0000:/,
       condition: function(data, matches) { return matches[1] == data.me; },
+      //this is fine in German. comment just for confirmation and can be deleted
```
Do you think it would be clearer to just return an object that said like {en: 'Orb Marker', de: 'Orb Marker'} even if they were the same, just to be obvious that the translation was complete?

Feel free to leave these sort of transitory comments to me as code review by just clicking on the line and leaving it (like I'm leaving this comment).  At the very least, go ahead and remove this comment in code before I merge this.

quisquous replied:
Ok! Could you remove this comment here and do that in this patch?

ArkfrostLumas replied:
The comment should already be removed and the new stuff pushed o.o

quisquous replied:
It still looks like it's in the patch.  If I look at 758099 then it's only changes to the byakko file.  If this is changed for you locally, can you make sure it's committed and pushed?

quisquous commented on ui/raidboss/data/triggers/o8s.js:
```diff
@@ -148,14 +148,14 @@
       id: "O8S Future's End",
       regex: /Kefka starts using Future's End/,
-      regexDe: /Kefka starts using Ende Der Hoffnung/,
+      regexDe: /Kefka starts using Ende Der Zukunft/,
```
That's funny that German was so different but then they changed it.  That also has to be confusing though if you've been clearing o8s for weeks and then suddenly the ability names aren't the same anymore.

quisquous commented on ui/raidboss/data/triggers/byakko-ex.js:
```diff
@@ -6,67 +6,119 @@
-          return 'Tank Buster on YOU';
-        if (data.role == 'healer')
-          return 'Buster on ' + data.ShortName(matches[1]);
+        if (matches[1] == data.me){
```
Sorry, can you add a space before the curly braces here and elsewhere in this file?

ArkfrostLumas replied:
Sure, no problem ^^

quisquous commented on ui/raidboss/data/triggers/byakko-ex.js:
```diff
@@ -6,67 +6,119 @@
       tts: function(data, matches) {
         if (matches[1] == data.me)
-          return 'buster';
+        return {
```
Can you indent this and add curly braces here too?

quisquous commented on ui/raidboss/data/triggers/sohm_al.js:
```diff
@@ -8,38 +8,70 @@
       regex: /1B:........:(\y{Name}):....:....:0017:0000:0000:0000:/,
       alertText: function(data, matches) {
         if (data.matches[1] == data.me)
```
Curly braces here on all these blocks and in the rest of this file.

quisquous replied:
It looks like you missed this.

quisquous commented:
Sorry for another round of back and forth.  I think there's just three more small things:
(1) you added some syntax errors in byakko in your last patch
(2) it looks like you missed the comment about sohm_al.js (adding curly braces) in the previous round of reviews
(3) you said o7s got changed, but looking at your last change and in your repository it still has the "this is fine in german" line that you said should be removed.  So, maybe something got missed!

ArkfrostLumas commented:
(1) Oh, didn't know the curly braces were supposed to be that way round. Corrected that now.

(2) Oh, I thought that comment was in byakkoEx only. Changed. Is it alright now?

(3) That's weird. On my local files it was already changed and I already had pushed it, but for some reason github didn't update it. Had to make another change (changed the o in orb to O) so git accepted the change.

quisquous commented:
Heck yeah, thanks for all this work! Since this includes the byakko and eureka stuff, I'll just close the other.
